src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/IsBlankOrError.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using Microsoft.PowerFx.Core.Functions;
+using Microsoft.PowerFx.Core.Localization;
+using Microsoft.PowerFx.Core.Texl.Builtins;
+using Microsoft.PowerFx.Core.Types;
+
+namespace Microsoft.PowerFx.Core.Texl.Builtins
+{
+    // IsBlankOrError(expression:E)
+    internal sealed class IsBlankOrErrorFunction : IsBlankFunctionBase
```

Comment:
Can you add tests for this function? Something along the lines of

    >> IsBlankOrError(If(1 < 0, "hello"))
    true

    >> IsBlankOrError("hello")
    false

    >> IsBlankOrError(If(1 / 0 < 2, "hello"))
    true